jupyterhub_singleuser_profiles/ui/src/ImageForm/ImageTagPopover.tsx

```diff
@@ -31,7 +34,7 @@ const ImageTagPopover: React.FC<ImageTagPopoverProps> = ({ tag, description }) =
               {description}
             </span>
           ) : null}
-          {tag.content?.dependencies?.length > 0 ? (
+          {tag?.content && tag.content.dependencies?.length > 0 ? (
```

Comment:
parallel: since "dependencies" is checked more than once, could "destructure" it around line 13 and carry it through... or set a variable... so many options =)

```
const { dependencies } = tag?.cont...
```